dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/service/FdcTestDataCreatorService.java

```diff
@@ -98,15 +98,9 @@ private void processNegativeTests(
           final FdcTestType testType, final Integer repOrderId, final Integer fdcId, final PickupType pickupType) {
     switch (testType) {
       case NEGATIVE_SOD -> {
-        switch (pickupType) {
-          case DELAYED_PICKUP -> {
-            Map<String, Object> repOrderWithNullSOD = new HashMap<>();
-            repOrderWithNullSOD.put("sentenceOrderDate", null);
-            testDataClient.updateRepOrderSentenceOrderDateToNull(repOrderId, repOrderWithNullSOD);
-          }
-          case FAST_TRACK -> testDataClient.updateRepOrderSentenceOrderDate(UpdateRepOrder.builder()
-              .repId(repOrderId).sentenceOrderDate(LocalDate.now().minusMonths(7)).build());
-        }
+        Map<String, Object> repOrderWithNullSOD = new HashMap<>();
```

Comment:
SOD is to be set to null for both Delayed and Fast Track pickups, which I didn't realise initially.